docs/CONTRIBUTING.md

```diff
@@ -28,6 +28,9 @@ An easier solution would be to use Docker.
 
 **Setup**
 1. Navigate to the root directory of telescope.
+1. Create a new file and name it ".env".
+1. Copy the contents of "env.example" into ".env".
```

Comment:
Combine these first 2 lines into one: `1. Copy env.example to .env to create a new environment configuration`

Reply:
Combined first 2 lines into one